src/main/java/com/ericsson/gerrit/plugins/eiffel/storage/EventStorage.java

```diff
@@ -49,24 +49,25 @@ protected String getLastSavedEiffelEvent(String project, String searchCriteria,
         return getEventId(project, searchCriteria, tableName);
     }
 
-    protected void findAndUpdateEiffelEvent(String project, String searchCriteria, String eiffelEvent, Table tableName)
+    protected void saveEiffelEventId(String project, String searchCriteria, String eiffelEvent, Table tableName)
             throws NoSuchElementException, ConnectException, SQLException {
         DatabaseHandler dBHandler = new DatabaseHandler(pluginDir, project);
-
-        saveEventToDatabase(project, searchCriteria, eiffelEvent, tableName, dBHandler);
+        String oldEventId = getOldEventId(dBHandler, tableName, searchCriteria);
```

Comment:
Please move get old event ID into saveOrUpdate, max recommended number of input variables is already exceeding 3.

Reply:
Changed in 23f9e70